controllers/compassmanager_controller.go

```diff
@@ -20,37 +22,44 @@ import (
 )
 
 const (
-	KymaNameLabel         = "operator.kyma-project.io/kyma-name"
-	CompassIDLabel        = "operator.kyma-project.io/compass-id"
-	BrokerPlanIDLabel     = "kyma-project.io/broker-plan-id"
-	BrokerPlanNameLabel   = "kyma-project.io/broker-plan-name"
-	GlobalAccountIDLabel  = "kyma-project.io/global-account-id"
-	BrokerInstanceIDLabel = "kyma-project.io/instance-id"
-	ShootNameLabel        = "kyma-project.io/shoot-name"
-	SubaccountIDLabel     = "kyma-project.io/subaccount-id"
+	KymaNameLabel                  = "operator.kyma-project.io/kyma-name"
+	BrokerPlanIDLabel              = "kyma-project.io/broker-plan-id"
+	BrokerPlanNameLabel            = "kyma-project.io/broker-plan-name"
+	GlobalAccountIDLabel           = "kyma-project.io/global-account-id"
+	BrokerInstanceIDLabel          = "kyma-project.io/instance-id"
+	ShootNameLabel                 = "kyma-project.io/shoot-name"
+	SubaccountIDLabel              = "kyma-project.io/subaccount-id"
+	KymaIDLabel                    = "kyma-project.io/kyma-id"
+	ComppassIDLabel                = "kyma-project.io/compass-runtime-id"
+	DefaultResourceNamespace       = "kcp-system"
+	ApplicationConnectorModuleName = "application-connector-module"
 	// KubeconfigKey is the name of the key in the secret storing cluster credentials.
 	// The secret is created by KEB: https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/internal/process/steps/lifecycle_manager_kubeconfig.go
 	KubeconfigKey = "config"
 )
 
 //+kubebuilder:rbac:groups=apiextensions.k8s.io,resources=customresourcedefinitions,verbs=get;list;watch
-//+kubebuilder:rbac:groups=operator.kyma-project.io,resources=kymas,verbs=get;list;watch;update
+//+kubebuilder:rbac:groups=operator.kyma-project.io,resources=kymas,verbs=get;list;watch
+//+kubebuilder:rbac:groups=operator.kyma-project.io,resources=compassmanagermappings,verbs=create;get;list;watch;update;delete
```

Comment:
The `watch` privilege is required. Because the controller returns an error
```
[reflector.go:148] pkg/mod/k8s.io/client-go@v0.27.3/tools/cache/reflector.go:231: Failed to watch *v1beta1.CompassManagerMapping: unknown (get compassmanagermappings.operator.kyma-project.io)
```